src/components/feed/NodeDetails/NodeDetails.tsx

```diff
@@ -94,6 +99,14 @@ const NodeDetails: React.FC<INodeProps> = ({ expandDrawer }) => {
     instanceParameters,
     pluginParameters,
   ]);
+  
+  const path = selectedPlugin?.collection.items[0].links.find((obj:any) => (obj.rel === "files")).href
```

Comment:
the variable name here is a little misleading. It's not a path, it's an API URI right?